Broadcast-Ext/Broadcast-Ext.xcodeproj/project.pbxproj

```diff
@@ -660,19 +690,23 @@
 				SDKROOT = iphoneos;
 				TARGETED_DEVICE_FAMILY = "1,2";
 				VALIDATE_PRODUCT = YES;
+				VALIDATE_WORKSPACE = YES;
 			};
 			name = Release;
 		};
 		D442612A185F81D800E23699 /* Release */ = {
 			isa = XCBuildConfiguration;
+			baseConfigurationReference = 44BD9C60A4EFC73599B43638 /* Pods-Broadcast-Ext.release.xcconfig */;
 			buildSettings = {
+				ALLOW_TARGET_PLATFORM_SPECIALIZATION = YES;
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				ASSETCATALOG_COMPILER_LAUNCHIMAGE_NAME = LaunchImage;
 				CODE_SIGN_ENTITLEMENTS = "Broadcast-Ext/Broadcast-Ext.entitlements";
 				CODE_SIGN_IDENTITY = "iPhone Developer";
 				CODE_SIGN_STYLE = Manual;
 				DEVELOPMENT_TEAM = "";
 				ENABLE_BITCODE = NO;
+				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "arm64 i386";
```

Comment:
@IGitGotIt @jvalli please provide feedback. I was facing the following issue 

> /Users/gmendes/samples/opentok-ios-sdk-samples/Broadcast-Ext/Broadcast-Ext.xcodeproj: error: The linked framework 'Pods_Broadcast_Ext_OpenTok_Live.framework' is missing one or more architectures required by this target: i386. (in target 'OpenTok Live' from project 'Broadcast-Ext')

, that was solved by excluding i386 architecture (arm64 was already there when I added it. Don't know why is not appearing). It does not look very good to me. What is your opinion?